moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -950,114 +947,77 @@ as the new values that correspond to the group */
      set by computing inverse kinematics.
       The pose is assumed to be in the reference frame of the kinematic model. Returns true on success.
       @param pose The pose the last link in the chain needs to achieve
+      @param attempts The number of times IK is attempted
       @param timeout The timeout passed to the kinematics solver on each attempt
       @param constraint A state validity constraint to be required for IK solutions */
-  bool setFromIK(const JointModelGroup* group, const geometry_msgs::Pose& pose, double timeout = 0.0,
-                 const GroupStateValidityCallbackFn& constraint = GroupStateValidityCallbackFn(),
+  bool setFromIK(const JointModelGroup* group, const geometry_msgs::msg::Pose& pose, unsigned int attempts = 0,
```

Comment:
@henningkayser can you help addressing this? what's your suggestion? is this something that will get fixed once we _sync_ with moveit1 master? If so, shall we skip your comment for now?

Reply:
Thanks @henningkayser, that makes sense. I recall @rhaschke pushing some stuff accidentally so it may be from that time since this PR is from a while ago.

I've created a new PR at https://github.com/ros-planning/moveit2/pull/59. Can you please have a look there?